api/app/graphql/mutations/create_task.rb

```diff
@@ -2,10 +2,13 @@ module Mutations
   class CreateTask < BaseMutation
     field :task, Types::TaskType, null: false
 
-    argument :name, String, required: true
+    argument :name, String, required: false, default_value: 'New Task'
+    argument :complete, Boolean, required: false, default_value: false
+    argument :user_id, ID, required: false, default_value: 1
```

Comment:
This is basically saying "If I don't specify the user, assume it belongs to the first user".

Totally fine in the context of this app and in the spirit of "getting it to work", but worth calling out, you probably would not want this behavior in a production application.

Reply:
💯 !!!

I actually did this on purpose since I didn't implement authentication, and actually originally has this on the frontend, but thought it was cleaner here instead. If we did have authentication I'd love to make it so the user that created the task was automatically made the owner